src/Microsoft.AspNetCore.Sockets/ConnectionManager.cs

```diff
@@ -135,6 +138,7 @@ public void Scan()
                     // Once the decision has been made to dispose we don't check the status again
                     if (status == DefaultConnectionContext.ConnectionStatus.Inactive && (DateTimeOffset.UtcNow - lastSeenUtc).TotalSeconds > 5)
                     {
+                        SocketEventSource.Log.ConnectionTimedOut(c.Value.ConnectionId);
```

Comment:
Where is the log for this?

Reply:
Yeah, we aren't doing the "bundle the event source call in the logger extension method" thing, but in general, it should be abnormal to write an event and **not** write a log message. So if there isn't a log message for this, add one as well :).